data_measurements/dataset_statistics.py

```diff
@@ -290,7 +290,6 @@ def __init__(
         self.node_list_fid = pjoin(self.cache_path, "node_list.th")
         # Needed for UI
         self.fig_tree_json_fid = pjoin(self.cache_path, "fig_tree.json")
-        self.load_or_prepare_dataset()
```

Comment:
Was this necessary?

It looks like it was added recently here: https://github.com/huggingface/data-measurements-tool/commit/033d285e2cfec4568423320f165f71c2955148ad#diff-4ef4d32c1293f3d378d897d2c0cc267c4488c0d689d334606d7078d239dbed90R311

load_or_prepare_dataset() is also called outside of the constructor, so not sure if this addition was intentional?

Reply:
If we leave it in, then it breaks my PR as it is currently implemented

Reply:
Ah, no it is not necessary!  Change it back, yes yes!

Reply:
Oh I think I wasn't very clear. I mean that I'm trying to delete this line in my PR. If I don't delete this line, then it breaks my PR. The line was very recently added, and I'm not sure why